controllers/comment.controller.js

```diff
@@ -0,0 +1,12 @@
+const { removeCommentById } = require('../models/comment.model');
+
+exports.deleteCommentById = async (req, res, next) => {
+  const { comment_id } = req.params;
+
+  try {
+    await removeCommentById(comment_id);
+    res.status(204).end();
```

Comment:
.end() doesn't set the content type and doesn't set a header, I think I would use .send() even if it's nothing being sent